x11-plugins/wmDownload/files/wmDownload-strtouq-musl.patch

```diff
@@ -0,0 +1,24 @@
+https://bugs.gentoo.org/898922
+From: Brahmajit Das <brahmajit.xyz@gmail.com>
+Date: Sun, 14 May 2023 14:39:59 +0530
+Subject: [PATCH] Don't use strtouq while building on non-glibc system
+
+Bug: https://bugs.gentoo.org/898922
+Signed-off-by: Brahmajit Das <brahmajit.xyz@gmail.com>
+--- a/wmDownload.c
++++ b/wmDownload.c
+@@ -205,7 +205,11 @@ void checknetstats ()
+ 				{
+ 					p = (char *)strtok (NULL, tokens);
+ 						
++#ifdef __GLIBC__
+ 					bytes = strtouq(p, NULL, 0);
++#else
++					bytes = strtoul(p, NULL, 0);
```

Comment:
I've not checked this, but googling came up with: http://lists.busybox.net/pipermail/buildroot/2016-May/160719.html.

We need to verify it's safe to replace it.

Reply:
Following that thread:

On a 64-bit arch, long is normally 64-bit, so strtouq is equivalent to strtoul.

  But normally strtoull should always exist for gcc, except when an explicit 
-std=c90 is passed.